deployment/postgres-init-db/sql/ui/tables/extensions.sql

```diff
@@ -0,0 +1,36 @@
+create table extensions (
```

Comment:
Doesn't belong in this first iteration, but would it be useful to store some input / output type definitions in the DB? That way, users could define a sort of custom API between the UI and their extensions, instead of our current hardcoded `ExtensionPayload`.

Reply:
This is a great idea, we'll see if Clipper has a use-case that we're not accounting for with our initial pass but maybe this could be an enhancement.

Reply:
I'm not against this proposal, but how would the UI know how to handle arbitrary custom inputs/outputs from extensions? Or is the thought that there is a set of things an extension can ask for/return, rather than the current "the UI will always hand the entire payload"?

Reply:
Interesting conversation we can have later if there is a need, doesn't need to block this PR though.